angular-workspace/projects/ni/nimble-angular/pipes/public-api.ts

```diff
@@ -1,2 +0,0 @@
-export * from './duration.pipe';
```

Comment:
We can make them standalone modules, that's fine, but we still need to export them from an entrypoint. I assume we still want to export them from `@ni/nimble-angular/pipes` as right now they seem like dead code not exported from anywhere

Reply:
We also add all components to the example app which exercises the entrypoint but that doesn't make as much sense for the pipes. Maybe the example app should have smoke tests that import the pipes to exercise the entrypoint behavior.

Reply:
Why doesn't it make sense to add the pipes to the example app? Are we unwilling to break the symmetry between the Angular and Blazor example apps?